app/models/consultee.rb

```diff
@@ -37,7 +37,7 @@ def period(now = Time.current)
   end
 
   def consulted?
-    awaiting_response? || responses?
+    !not_consulted?
```

Comment:
I don't like the "not-not-consulted", but I presume that as `not_consulted` comes from the enum there's nothing much to do about this.